document_merge_service/settings.py

```diff
@@ -235,9 +237,25 @@ def parse_admins(admins):
             "level": "WARNING",
             "filters": None,
             "class": "logging.StreamHandler",
-        }
+        },
+        "mail_admins": {
+            "level": "ERROR",
+            "filters": None,
+            "class": "django.utils.log.AdminEmailHandler",
+        },
     },
-    "loggers": {"django": {"handlers": ["console"], "level": "WARNING"}},
+    "loggers": {"django": {"handlers": ["console", "mail_admins"], "level": "WARNING"}}
+    if ENABLE_ADMIN_EMAIL_LOGGING
+    else {"django": {"handlers": ["console"], "level": "WARNING"}},
```

Comment:
Maybe just put the extra handler in the if block below:

```python
if ENABLE_ADMIN_EMAIL_LOGGING:
    LOGGING["loggers"]["handlers"].append("mail_admins")
```

This will make it better readable and we avoid redundancy

Reply:
`mypy` can't check for nested `dicts`, that's why I added `# type: ignore`